packages/api/AlvTime.Persistence/Repositories/FlexhourStorage.cs

```diff
@@ -25,19 +26,14 @@ public AvailableHoursDto GetAvailableHours(int userId)
         var startDate = dbUser.StartDate;
         var endDate = DateTime.Now.Date;
 
-        var timeOff = _context.Hours.Where(h => h.User == userId && h.TaskId == 18).Sum(h => h.Value);
-        var hoursWorkedOver = GetHoursWorkedMoreThanWorkday(startDate, endDate, userId).Sum(e => e.Value) - timeOff;
-
-        var hoursWithCompRate = GetOvertimeEquivalents(startDate, endDate, userId);
-
         var entriesByDate = GetTimeEntries(startDate, endDate, userId);
-
-        var overtimeEntries = GetOvertimeEntries(entriesByDate, startDate, endDate);
+        var overtimeEntries = GetOvertimeEntries(entriesByDate, startDate, endDate);
+        (var overtime, var flex) = GetFlexAndOvertime(startDate, endDate, userId);
```

Comment:
Supert at du fikk fjernet duplikat logikk for å beregne fleks- og overtidstimer. Jeg har fortsatt en magefølelse av at vi kan forenkle ting ytterligere, men det kan vi spare til runde to!